lib/cfn-model/parser/ec2_network_acl_parser.rb

```diff
@@ -8,45 +8,23 @@
 class Ec2NetworkAclParser
   def parse(cfn_model:, resource:)
     network_acl = resource
-
     attach_nacl_entries_to_nacl(cfn_model: cfn_model, network_acl: network_acl)
     network_acl
   end
 
   private
 
-  def egress_network_acl_entries(cfn_model)
-    network_acl_entries = cfn_model.resources_by_type 'AWS::EC2::NetworkAclEntry'
-    network_acl_entries.select(&:egress)
-  end
-
-  def ingress_network_acl_entries(cfn_model)
-    network_acl_entries = cfn_model.resources_by_type 'AWS::EC2::NetworkAclEntry'
-    network_acl_entries.select do |network_acl_entry|
-      not_truthy?(network_acl_entry.egress)
-    end
-  end
-
-  def egress_nacl_entries_for_nacl(cfn_model, logical_resource_id)
-    egress_nacl_entries = egress_network_acl_entries(cfn_model)
-    egress_nacl_entries.select do |egress_nacl_entry|
-      References.resolve_resource_id(egress_nacl_entry.networkAclId) == logical_resource_id
-    end
-  end
-
-  def ingress_nacl_entries_for_nacl(cfn_model, logical_resource_id)
-    ingress_nacl_entries = ingress_network_acl_entries(cfn_model)
-    ingress_nacl_entries.select do |ingress_nacl_entry|
-      References.resolve_resource_id(ingress_nacl_entry.networkAclId) == logical_resource_id
+  def nacl_entries_for_nacl(cfn_model, logical_resource_id)
+    network_acl_entries = cfn_model.resources_by_type('AWS::EC2::NetworkAclEntry')
```

Comment:
small nit but you can just have the select() call here.  the value of the last statement in a method becomes the return value